tests/integration/docker/cp.bats

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bats
+# *-*- Mode: sh; sh-basic-offset: 8; indent-tabs-mode: nil -*-*
+
+#  This file is part of cc-oci-runtime.
+#
+#  Copyright (C) 2016 Intel Corporation
+#
+#  This program is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU General Public License
+#  as published by the Free Software Foundation; either version 2
+#  of the License, or (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program; if not, write to the Free Software
+#  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+#Based on docker commands
+
+SRC="${BATS_TEST_DIRNAME}/../../lib/"
+
+setup() {
+	source $SRC/test-common.bash
+	clean_docker_ps
+	runtime_docker
+}
+
+@test "Check mounted files at /etc after a docker cp" {
+	skip "currently this test is not working (see https://github.com/01org/cc-oci-runtime/issues/325)"
+	content="test"
+	$DOCKER_EXE run --name containertest -d ubuntu bash -c "sleep 50"
+	echo $content > testfile
```

Comment:
What's the working directory when this code is run? I'd be tempted to do something like this to guard against running in an environment where the source directory may not be writable (maybe an out-of-tree build-type environment):

```
testfile=$(mktemp --tmpdir="$BATS_TMPDIR" --suffix=-cor-test)
echo "$content" > "$testfile"
     :
     :
rm -f "$testfile"
```

If you don't like the fact that you are creating, then rewriting the temp file, you could use:

```
test_dir=$(mktemp -d --tmpdir="$BATS_TMPDIR" --suffix=-cor-test)
echo "$content" > "$test_dir/$testfile"
     :
     :
rm -rf "$test_dir"
```